### PR TITLE
Warn on deleting cluster with a user prompt.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
## Fixes / Features
- Warn on deleting cluster with a user prompt.

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Comment by mbobrovskyi** [comment]
cc: @pawloch00 @IrvingMg

File: src/xpk/commands/cluster.py [hunk 1]
```diff
@@ -423,6 +425,23 @@ def run_gke_cluster_delete_command(args) -> int:
...
+    xpk_print('Get the name of the workloads in the cluster.')
+    args.filter_by_status = 'EVERYTHING'
+    return_code, return_value = get_workload_list(args)
+    if return_code != 0:
+      xpk_print(f'List Job request returned ERROR {return_code}')
+      return return_code
+
+    workloads = [x.split(' ')[0] for x in return_value.splitlines()][1:]
+    if workloads and not get_user_input(
+        f'Planning to delete {len(workloads)} workloads in the cluster'
```
Comment (pawloch00) [inline src/xpk/commands/cluster.py:438]: iiuc this means deleting all workflows? If so please add this info

Comment (mbobrovskyi) [reply src/xpk/commands/cluster.py:438]:
```suggestion
        f'Planning to delete all ({len(workloads)}) workloads in the cluster'
```
Like this?

Comment (mbobrovskyi) [reply src/xpk/commands/cluster.py:438]: Or just

```suggestion
        f'Planning to delete all workloads in the cluster'
```

?

Comment (pawloch00) [reply src/xpk/commands/cluster.py:438]: yep

**Review by IrvingMg — DISMISSED** [review]

**Comment by mbobrovskyi** [comment]
cc: @pawloch00

File: src/xpk/commands/cluster.py [hunk 2]
```diff
@@ -423,6 +425,23 @@ def run_gke_cluster_delete_command(args) -> int:
...
   """
+  if not args.force:
+    xpk_print('Get the name of the workloads in the cluster.')
+    args.filter_by_status = 'EVERYTHING'
+    return_code, return_value = get_workload_list(args)
+    if return_code != 0:
+      xpk_print(f'List Job request returned ERROR {return_code}')
+      return return_code
+
+    workloads = [x.split(' ')[0] for x in return_value.splitlines()][1:]
```
Comment (pawloch00) [inline src/xpk/commands/cluster.py:436]: what is the output? will workloads be all printed in one line?

Comment (pawloch00) [reply src/xpk/commands/cluster.py:436]: If so, it would be nice to have them listed in new lines

**Review by pawloch00 — DISMISSED** [review]

File: src/xpk/commands/cluster.py (hunk 2 above)
Comment (mbobrovskyi) [reply src/xpk/commands/cluster.py:436]:
```
Planning to delete 3 workloads in the cluster xpk-test including ['sample-job-kmr9w', 'sample-job-rbkkw', 'sample-job-v7l9r']. 
Do you wish to delete: y (yes) / n (no):
```

That what we have.

Comment (mbobrovskyi) [reply src/xpk/commands/cluster.py]: I think it's okay because the terminal should automatically justify it.

**Review by pawloch00 — APPROVED** [review]

**Pull request merged**